test/testdrive/numeric.td

```diff
@@ -37,7 +37,7 @@ a
 # statement ok
 > CREATE OR REPLACE MATERIALIZED VIEW numeric_cast_ok AS
   SELECT 1
-  WHERE mz_logical_timestamp() > 1927418240000::numeric(38,0);
```

Comment:
What caused these changes?

Reply:
Previously `mz_logical_timestamp()` would return `Timestamp::MAX` in a view which made this true. Now it returns the current timestamp of the timeline it's in making it false.